aip/0140.md

```diff
@@ -48,45 +48,50 @@ together.
 ### Repeated fields
 
 Repeated fields **must** use the proper plural form, such as `books` or
-`authors`.
+`authors`. On the other hand, non-repeated fields **should** use the
+singular form such as `book` or `author`. This implies that resource names
+**should** use the singular form as well as the field name should follow
```

Comment:
I don't understand this sentence at all, I'm afraid.

Reply:
Missing a comma. It implies that resource names should be singular as well -- because the field name should follow the message name. We've seen this more than I'd hoped to see it, where folks do something like ... `message Books { ... }` which becomes `Books books` as a field... It should be `message Book { ... }` and then `repeated Book books` as the field.